project-template.md

```diff
@@ -20,6 +20,10 @@ In general, OTEPs are not accepted unless they come with working prototypes avai
 
 Who is currently planning to work on the project? If a project requires specialized domain expertise, please list it here. If a project is missing a critical mass of people in order to begin work, please clarify.
 
+### People to contact
+
+Who (people, companies) in the industry should be aware of this effort? Was there an attempt to get them onboard? What did they say?
```

Comment:
Once the explanatory text is removed, "People to contact" may look like a list of people to contact within the SIG if someone has a question, which I don't think is the purpose of this change.

What are your thoughts on something like "Industry outreach" or "Consultation and feedback"?

Reply:
Great point. I'll rename the section.